### PR TITLE
:sparkles: Add host refresh operation

### PR DESCRIPTION
**Pull request by fanshan**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1 +1 @@
-{"netbox_url":"","netbox_token":"","ui":1,"id":"","endpoint":"","flows":"0"}
```
Comment (lvenier) [inline config.js:1]: why uploading the config.js ?

Comment (fanshan) [reply config.js:1]: Because it's not in the .gitignore file?

**Review by lvenier — APPROVED** [review]

**Review by linkdd — CHANGES_REQUESTED** [review]
Maybe remove the `debug` nodes?

Should we also consider using more "sub flows"? The `API - POST` flow is getting out of hand (spreading in 2 directions: left and bottom).

**Comment by fanshan** [comment]
> Maybe remove the `debug` nodes?
> 
> Should we also consider using more "sub flows"? The `API - POST` flow is getting out of hand (spreading in 2 directions: left and bottom).

Done for debug node.

I think some housekeeping for flows redability should be done ;-)

**Review by linkdd — APPROVED** [review]

**Pull request merged**